### PR TITLE
MHR Correction Framework

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#19501

*Description of changes:*
* Corrections Framework - Initialize Corrections (staff and client) - store snapshot - parse current data into UI.
* MHR Global Changes: Re-orders Steps and some components within steps
* refactors/renames TombstoneDischarge to more generic TombstoneDynamic (opposed to the static version)
* Removes overflow hidden on tables
* Minor FeeSummary updates for STAFF Corrections (Will do Client Fee addition during Client tickets)
* Breadcrumb updates to support Correction flow
* Implements Feature Flagged Correction Launch Btns (Hardcoded disabled Public Amendments for now)
* Mhr Validation updates to support step/component re-order
* Mhr Correction Composable
* Tombstone and Mhr Unit Test updates

Screenshot 2024-02-26 at 6 58 23 PM

Screenshot 2024-02-26 at 6 58 40 PM

Screenshot 2024-02-26 at 6 58 59 PM

*Issue #:* /bcgov/entity#19510
* Updates to Include ability to change MHR Status
* MHR status change component
* Unit tests
* Fixed typing on UpdateBadge prop type

Screenshot 2024-02-27 at 10 56 54 AM

Screenshot 2024-02-27 at 10 57 01 AM

Screenshot 2024-02-27 at 10 57 08 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1752-ywi8gi0d.web.app

**Review by doug-lovett — APPROVED** [review]
Looks good.

File: ppr-ui/src/views/newMhrRegistration/SubmittingParty.vue [hunk 1]
```diff
@@ -87,6 +73,20 @@
...
+    <section
+      id="mhr-add-submitting-party"
+      class="mt-10"
+    >
+      <ContactInformation
+        :contactInfo="getMhrRegistrationSubmittingParty"
+        :content="submittingPartyRegistrationContent"
+        :validate="validateSubmitter"
+        :sectionNumber="2"
+        @set-store-property="setMhrRegistrationSubmittingParty"
```
Comment (dimak1) [inline ppr-ui/src/views/newMhrRegistration/SubmittingParty.vue]: CamelCase here?

**Review by dimak1 — APPROVED** [review]
Looks great, just one comment.

File: ppr-ui/src/views/newMhrRegistration/SubmittingParty.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/views/newMhrRegistration/SubmittingParty.vue]: Good catch, thank you!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1752-ywi8gi0d.web.app

**Pull request merged**